plugins/urls/handlers/website.py

```diff
@@ -0,0 +1,266 @@
+__author__ = 'Gareth Coles'
+
+import os
+import re
+import socket
+
+from bs4 import BeautifulSoup
+from cookielib import LoadError
+from kitchen.text.converters import to_unicode, to_bytes
+from netaddr import IPAddress
+from twisted.web._newclient import ResponseNeverReceived
+from txrequests import Session
+
+from plugins.urls.handlers.handler import URLHandler
+from plugins.urls.cookiejar import ChocolateCookieJar
+from utils.misc import str_to_regex_flags
+
+
+class WebsiteHandler(URLHandler):
+    name = "website"
+
+    criteria = {
+        "protocol": re.compile(r"http|https", str_to_regex_flags("iu"))
+    }
+
+    group_sessions = {}
```

Comment:
`criteria` is borderline, since it's pretty much a constant. `group_sessions` though is definitely an instance value, so should go in `__init__`.